src/mumble/ServerHandler.cpp

```diff
@@ -454,6 +462,8 @@ void ServerHandler::message(unsigned int msgType, const QByteArray &qbaMsg) {
 			ConnectionPtr connection(cConnection);
 			if (!connection) return;
 
+			iInFlightTCPPings -= 1;
```

Comment:
Resetting rather than decrementing (as discussed in IRC) would make this "asymmetric decrement" less dangerous/problematic.